build/init/fetch-linux-headers.sh

```diff
@@ -27,7 +27,7 @@ generate_headers()
 }
 
 fetch_cos_linux_sources()
-{
+{S
```

Comment:
```suggestion
{
```

This seems like a typo @yuvipanda!